enterprise/migrations/0041_auto_20180212_1507.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import multi_email_field.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0040_auto_20180129_1428'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='enterprisecustomerreportingconfiguration',
+            name='email',
+            field=multi_email_field.fields.MultiEmailField(help_text='The email(s), one per line, where the report should be sent.', verbose_name='Email', blank=True),
```

Comment:
Does this migration change the underlying database column type of the existing `email` field? I ask because that could be a problem when trying to deploy this change to stage/prod due to the way that we apply migrations in those environments.

Reply:
@douglashall Good question. Guess I need to learn more about how we do migrations :) As for the `email` field, the `django-multi-email-field` stores data as multi-line text (with newline characters).

The underlying database column types of the `email` field locally (after the migration) and on production (form the read replica) are both `varchar(254)`, so I don't believe there is a change to the column type.

FWIW, maybe we can touch base on how we apply migrations in stage/prod?

Reply:
Okay, you should be good to go then.